vote/models.py

```diff
@@ -97,7 +97,8 @@ class Video(models.Model):
     game = models.CharField(max_length=200, blank=False, default="")
     author_name = models.CharField(max_length=200, blank=False, default="")
     video_link = models.CharField(max_length=200, blank=False, unique=True, validators=[URLValidator()])
-    follow_me_link = models.CharField(max_length=200, default="", blank=True, validators=[URLValidator()])
+    video_download_link = models.CharField(max_length=200, blank=False, unique=True, validators=[URLValidator()])
+    follow_me_link = models.CharField(max_length=200, default="", blank=True)
```

Comment:
Is it intended that the URL validator is removed for the follow me link? The PR does not say anything about it

Reply:
yes, forgot to mention it! I believe we discussed removing the validator there, but can be re-added and maybe done in a separate PR.

Reply:
no it's fine to remove it here, just wanted to make sure it's intentional